src/test/java/org/jabref/logic/layout/format/ToLowerCaseTest.java

```diff
@@ -19,7 +19,7 @@ public void testNull() {
 
     @Test
     public void testLowerCase() {
-        assertEquals("abcd efg", new ToLowerCase().format("abcd efg"));
+        assertEquals("abcd efg ", new ToLowerCase().format("abcd efg "));
```

Comment:
Please do not modify existing test cases. Please revert this change.

If you think, this is important, please add another test case.

Reply:
I've included it in the parameterized test now, is that OK or shall I revert it to its original form?